diff --git git/arch/arm/plat-omap/mcbsp.c-orig git/arch/arm/plat-omap/mcbsp.c
index 0c8612f..070c86a 100644
--- git/arch/arm/plat-omap/mcbsp.c-orig
+++ git/arch/arm/plat-omap/mcbsp.c
@@ -14,6 +14,7 @@
 
 #include <linux/module.h>
 #include <linux/init.h>
+#define DEBUG
 #include <linux/device.h>
 #include <linux/platform_device.h>
 #include <linux/wait.h>
@@ -1782,6 +1783,8 @@ static int __devinit omap_mcbsp_probe(struct platform_device *pdev)
 		goto err_ioremap;
 	}
 
+	dev_info(&pdev->dev, "mcbsp->phys_base: %p\n", mcbsp->phys_base);
+
 	/* Default I/O is IRQ based */
 	mcbsp->io_type = OMAP_MCBSP_IRQ_IO;
 	mcbsp->tx_irq = pdata->tx_irq;
@@ -1789,6 +1792,9 @@ static int __devinit omap_mcbsp_probe(struct platform_device *pdev)
 	mcbsp->dma_rx_sync = pdata->dma_rx_sync;
 	mcbsp->dma_tx_sync = pdata->dma_tx_sync;
 
+	dev_info(&pdev->dev, "mcbsp->dma_rx_sync: %u\n", mcbsp->dma_rx_sync);
+	dev_info(&pdev->dev, "mcbsp->dma_tx_sync: %u\n", mcbsp->dma_tx_sync);
+
 	mcbsp->iclk = clk_get(&pdev->dev, "ick");
 	if (IS_ERR(mcbsp->iclk)) {
 		ret = PTR_ERR(mcbsp->iclk);
